pkgs/build-support/setup-hooks/patch-shebangs.sh

```diff
@@ -118,7 +118,7 @@ patchShebangs() {
         newInterpreterLine="$newPath $args"
         newInterpreterLine=${newInterpreterLine%${newInterpreterLine##*[![:space:]]}}
 
-        if [[ -n "$oldPath" && ( "$update" == true || "${oldPath:0:${#NIX_STORE}}" != "$NIX_STORE" ) ]]; then
+        if [[ -n "$oldPath" && ( "${update}" == true || "${oldPath:0:${#NIX_STORE}}" != "$NIX_STORE" ) ]]; then
```

Comment:
This change isn't necessary, is it?

Reply:
No, it isn't, forgot to undo it when changing to the other method. My bad